src/eth/monitor.ts

```diff
@@ -89,24 +92,28 @@ async function main() {
     },
   };
 
-  await withEventTracker(
-    config.deviceId,
-    BridgeJSON,
-    config.rpcSource,
-    config.monitorAccount,
-    config.mongodbUrl,
-    (eventTracker: EventTracker) => {
-      return eventTracker.syncEvents(
-        async (eventName: string, v: any, hash: string) => {
-          return (handlers as any)[eventName](v, hash);
-        }
-      );
-    }
-  );
-
-  const warningAmount = "1";
-  await checkDeployerAccountBalance(config, warningAmount);
+  try {
+    await withEventTracker(
+      config.deviceId,
+      BridgeJSON,
+      config.rpcSource,
+      config.monitorAccount,
+      config.mongodbUrl,
+      (eventTracker: EventTracker) => {
+        return eventTracker.syncEvents(
+          async (eventName: string, v: any, hash: string) => {
+            return (handlers as any)[eventName](v, hash);
+          }
+        );
+      }
+    );
+  
+    const warningAmount = "1";
+    await checkDeployerAccountBalance(config, warningAmount);
 
+  } catch (e) {
+    sendAlert(e, SlackConfig, true);
```

Comment:
forget a throw there? (The old behaviour will pass exception outside but here your catch handles it silently.